packages/vue-instantsearch/src/components/Pagination.vue

```diff
@@ -88,6 +88,7 @@
             <a
               :class="suit('link')"
               :href="state.createURL(page)"
+              :aria-label="`Page ${page + 1}`"
```

Comment:
other aria-labels aren't configurable, so adding this one to add text isn't a breaking change